### DISCUSSION TITLE
Support for ebooks and audiobooks?

### DISCUSSION THREAD

**Discussion by ArbitraryAlex** [Komet/MediaElch#1720] [Ideas]
Pretty much self-explanatory but do you plan to support scraping for ebooks and audiobooks in the future? Some metadata providers could be Google Books, Open Library, iTunes and Audible.

**Comment by bugwelle** [comment]
Hi,

ebooks/audiobooks are out of scope of this project, sorry. 
See also: https://github.com/Komet/MediaElch/issues/1710

Regards,
Andre

**Reply by ArbitraryAlex** [reply]
Oh damn, okay. Thank you.
